### PR TITLE
Expand a Reuters.com ruleset

### LINKED ISSUE #10686: Reuters.com doesn't switch to HTTPS even though it's available
HTTPS Everywhere is enabled in Firefox, but if I go to reuters.com, it remains unencrypted. If I go to https://www.reuters.com it works.

This most likely has something to do with the www. prefix.

Could it be an idea to implement a list in the plug-in that contains special instructions to switch to https for some websites?

tl;dr HTTPS Everywhere does not enable HTTPS on reuters.com because merely changing http:// to https:// is not enough; it needs to change http:// to https://www..

### PR DESCRIPTION
**Pull request by koops76**
Closes #10686

### CODE REVIEW

File: src/chrome/content/rules/Reuters.com.xml [hunk 1]
```diff
@@ -59,6 +59,8 @@
 -->
 <ruleset name="Reuters.com (partial)">
 
+	<target host="reuters.com" />
+	<target host="www.reuters.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Reuters.com.xml]: Please also update the comments on top.
See also:
af.reuters.com
lta.reuters.com
ara.reuters.com
ar.reuters.com
br.reuters.com
ca.reuters.com
cn.reuters.com
de.reuters.com
es.reuters.com
fr.reuters.com
in.reuters.com
it.reuters.com
jp.reuters.com
mx.reuters.com
ru.reuters.com
uk.reuters.com

widerimage.reuters.com
agency.reuters.com

Mismatch:
live.reuters.com
mobile.reuters.com

Refused:
sales.reuters.com

File: src/chrome/content/rules/Reuters.com.xml [hunk 2]
```diff
@@ -77,9 +79,9 @@
 	<!--securecookie host="^customers\.reuters\.com$" name="^ASP\.NET_SessionId$" /-->
 	<!--securecookie host="^pictures\.reuters\.com$" name="^(?:ASP\.NET_SessionId|CorexLanguageRTR)$" /-->
 
-	<securecookie host="^\w" name="." />
-
+	<securecookie host=".+" name=".+" />
 
+	<rule from="^http://reuters\.com/" to="https://www.reuters.com/" />
```
Comment (J0WI) [inline src/chrome/content/rules/Reuters.com.xml]: Please document why this special redirect is required.

Comment (koops76) [inline src/chrome/content/rules/Reuters.com.xml]: https://reuters.com times out.

File: src/chrome/content/rules/Reuters.com.xml (hunk 1 above)
Comment (koops76) [inline src/chrome/content/rules/Reuters.com.xml]: Done.

Comment (J0WI) [inline src/chrome/content/rules/Reuters.com.xml]: I still miss all the [region].reuters.com subdomains.

Comment (koops76) [inline src/chrome/content/rules/Reuters.com.xml]: Do all of them work over HTTPS?

Comment (koops76) [inline src/chrome/content/rules/Reuters.com.xml]: Also what about widerimage.reuters.com and agency.reuters.com?

Comment (J0WI) [inline src/chrome/content/rules/Reuters.com.xml]: Yes, they WFM. Including `af` and `ca` which are currently listed as non-woring.

Comment (koops76) [inline src/chrome/content/rules/Reuters.com.xml]: Done.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**